8.5-java11/init_container.sh

```diff
@@ -48,6 +48,11 @@ fi
 if [ -z "$WEBSITE_INSTANCE_ID" ]
```

Comment:
May be a good idea to update the comment:
"WEBSITE_INSTANCE_ID will be defined uniquely for each worker instance while running in Azure.
During development it may not be defined, in that case  we set WEBSITE_INSTNACE_ID=dev."